capstone-frontend/src/components/SearchUserModal.jsx

```diff
@@ -122,7 +122,7 @@ export class SearchUserModal extends Component {
       add_collaboratorsIDs: user.id,
     }
 
-    // Add Collaborator to Booklist in Firebase
+    // Remove BookList in Firebase
```

Comment:
I don't think this comment accurately describes the code it is next to

Reply:
This was probably a merge error I overlooked.